src/Parser/DeathKnight/Unholy/Modules/Features/ScourgeStrikeEfficiency.js

```diff
@@ -0,0 +1,89 @@
+import React from 'react';
+
+import SPELLS from 'common/SPELLS';
+import SpellIcon from 'common/SpellIcon';
+import SpellLink from 'common/SpellLink';
+import { formatPercentage } from 'common/format';
+
+import Analyzer from 'Parser/Core/Analyzer';
+import Enemies from 'Parser/Core/Modules/Enemies';
+import Combatants from 'Parser/Core/Modules/Combatants';
+import StatisticBox, { STATISTIC_ORDER } from 'Main/StatisticBox';
+
+class ScourgeStrikeEfficiency extends Analyzer {
+  static dependencies = {
+    enemies: Enemies,
+    combantants: Combatants,
+  };
+
+  // used to track how many stacks a target has
+  targets = {};
+
+  totalScourgeStrikeCasts = 0;
+  scourgeStrikeCastsZeroWounds = 0;
+
+  on_byPlayer_applydebuffstack(event){
+    const spellId = event.ability.guid;
+    if(spellId === SPELLS.FESTERING_WOUND.id){
+      this.targets[event.targetID] = event.stack;
+    }
+  }
+
+  on_byPlayer_removedebuffstack(event){
+    const spellId = event.ability.guid;
+    if(spellId === SPELLS.FESTERING_WOUND.id){
+      this.targets[event.targetID] = event.stack;
+    }
+  }
+
+  on_byPlayer_removedebuff(event){
+    // keeps track of when the debuff drops off, useful for multitarget targets like DI
+    const spellId = event.ability.guid;
+    if(spellId === SPELLS.FESTERING_WOUND.id){
+      this.targets[event.targetID] = 0;
+    }
+  }
+
+  on_byPlayer_cast(event){
+    const spellId = event.ability.guid;
+    if(spellId === SPELLS.SCOURGE_STRIKE.id){
+      this.totalScourgeStrikeCasts++;
+      if(this.targets.hasOwnProperty(event.targetID)){
+        const currentTargetWounds = this.targets[event.targetID];
+        if(currentTargetWounds < 1){
+          this.scourgeStrikeCastsZeroWounds++;
+        }
+      }
+    }
+  }
+
+  suggestions(when) {
+    const percentCastZeroWounds = this.scourgeStrikeCastsZeroWounds/this.totalScourgeStrikeCasts;
+    const strikeEfficiency = 1 - percentCastZeroWounds;
+    when(strikeEfficiency).isLessThan(0.60)
+        .addSuggestion((suggest, actual, recommended) => {
+          return suggest(<span>You are casting <SpellLink id={SPELLS.SCOURGE_STRIKE.id}/> too often.  When spending runes remember to cast <SpellLink id={SPELLS.FESTERING_STRIKE.id}/> instead on targets with no stacks of <SpellLink id={SPELLS.FESTERING_WOUND.id}/></span>)
+            .icon(SPELLS.SCOURGE_STRIKE.icon)
+            .actual(`${formatPercentage(actual)}% of Scourge Strikes were used with Wounds on the target`)
+            .recommended(`>${formatPercentage(recommended)}% is recommended`)
+            .regular(recommended - 0.30).major(recommended - 0.40);
+        });
```

Comment:
This seems like a rather large jump from minor to average and then followed by a smaller jump from average to major.
This is your current scheme: >60% --> >30% --> >20%.
Wouldn't a 60 -> 40 -> 20 make more sense?

Reply:
Tacking onto that, I would raise the suggestion threshold to 80 or 85%.  Scourge Strike should almost never be cast without a Festering Wound on the target as bursting wounds provides significantly more damage than scourge strike itself.  There are occasional situations in burst AoE where you might spam SS on a target with no wounds but generally speaking we should see this number >90% when playing the rotation correctly.